### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by Thomas-Wallis**
Please check formatting, naming and logic @RyanQ02

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        if verify_name.upper() != "Y" and verify_name != "":
+            name = input("Name: ")
+        email_names[email] = name
+        email = input("Email: ")
+    for email, name in email_names.items():
+        print("{} ({})".format(name, email))
+
+
+def get_email_name(email):
+    """Takes excepted name from email"""
```
Comment (orth22) [inline prac_05/emails.py:17]: Good use of comments explaining what the function does

File: prac_05/hex_colours.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+HEX_COLOURS = {"aliceblue": "#f0f8ff", "antiquewhite": "faebd7", "antiquewhite1": "ffefdb", "antiquewhite2": "#eedfcc",
+               "antiquewhite3": "#cdc0b0", "antiquewhite4": "#8b8378", "aquamarine1": "#8b8378",
+               "aquamarine2": "#7fffd4", "aquamarine4": "#76eec6", "azure1": "#f0ffff", "azure2": "#e0eeee"}
+colour = input("Please enter the colour you would like to check the hexadecimal code of: ").lower()
+while colour != "":
```
Comment (orth22) [inline prac_05/hex_colours.py:5]: Good use of catching no input error

File: prac_05/word_occurrences.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+word_count = {}
+sentence = input("Enter words: ")
```
Comment (orth22) [inline prac_05/word_occurrences.py:2]: Could put a no input while loop here to catch when a user enters nothing here instead of the program breaking.

**Review by orth22 — CHANGES_REQUESTED** [review]

File: prac_05/word_occurrences.py (hunk 3 above)
Comment (Thomas-Wallis) [reply prac_05/word_occurrences.py:2]: Noted. I expect I overlooked this. Thank you

**Pull request merged**